platform/src/test/kotlin/org/stellar/anchor/platform/service/SimpleInteractiveUrlConstructorTest.kt

```diff
@@ -59,17 +69,25 @@ class SimpleInteractiveUrlConstructorTest {
     val testRequest = gson.fromJson(requestJson, HashMap::class.java)
     val testTxn = gson.fromJson(txnJson, JdbcSep24Transaction::class.java)
 
-    val constructor = SimpleInteractiveUrlConstructor(testConfig, customerIntegration, jwtService)
+    val constructor =
+      SimpleInteractiveUrlConstructor(clientsConfig, testConfig, customerIntegration, jwtService)
 
     var jwt =
       parseJwtFromUrl(constructor.construct(testTxn, testRequest as HashMap<String, String>?))
     testJwt(jwt)
+    var claims = jwt.claims
     assertEquals("GBLGJA4TUN5XOGTV6WO2BWYUI2OZR5GYQ5PDPCRMQ5XEPJOYWB2X4CJO:1234", jwt.sub)
+    assertEquals("lobstr.co", claims["client_domain"] as String)
+    assertEquals("lobstr", claims["client_name"] as String)
 
     testTxn.sep10AccountMemo = null
+    testTxn.clientDomain = "unknown.com"
     jwt = parseJwtFromUrl(constructor.construct(testTxn, testRequest))
+    claims = jwt.claims
     testJwt(jwt)
     assertEquals("GBLGJA4TUN5XOGTV6WO2BWYUI2OZR5GYQ5PDPCRMQ5XEPJOYWB2X4CJO", jwt.sub)
+    assertEquals("unknown.com", claims["client_domain"] as String)
+    assertEquals("", claims["client_name"] as String)
```

Comment:
Can you add test for custodial wallet? It won't have a client_domain in the challenge transaction

Reply:
Done